pkgs/by-name/ll/llama-cpp/package.nix

```
@@ -146,7 +156,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
     license = licenses.mit;
     mainProgram = "llama-cpp-main";
     maintainers = with maintainers; [ dit7ya elohmeier ];
```

You can add me (and @SomeoneSerge if he accepts 🙇🏻 ).

```suggestion
    maintainers = with maintainers; [ dit7ya elohmeier philiptaron ];
```